src/ml_tooling/transformers/pandas_transformers.py

```diff
@@ -206,3 +206,26 @@ def transform(self, X):
         for col in X.columns:
             X[col] = X[col].str.upper().map(self.frequencies[col]).fillna(0)
         return X
+
+
+class DFStandardScaler(BaseEstimator, TransformerMixin):
+    """
+    Implementation of the StandardScaler from scikit-learn for Pandas DataFrames
```

Comment:
Måske en link til sklearn dokumentationen? Jeg kommer nok til at gøre det samme for de andre også 😄